### PR TITLE
Support for custom validation error

### PR DESCRIPTION
**Pull request by dugenkui03**

### CODE REVIEW

File: src/main/java/graphql/validation/ValidationErrorTypeInterface.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package graphql.validation;
+
+
+import graphql.PublicApi;
+
+@PublicApi
+public interface ValidationErrorTypeInterface {
```
Comment (dugenkui03) [inline src/main/java/graphql/validation/ValidationErrorTypeInterface.java:7]: Mark interface for validation error type.

Comment (bbakerman) [reply src/main/java/graphql/validation/ValidationErrorTypeInterface.java:7]: I am happy to accept this PR - However I would like the name of this to be  `ValidationErrorClassification` (like how we have `ErrorClassification`

**Review by bbakerman — CHANGES_REQUESTED** [review]
Can we rename to `ValidationErrorClassification` please

**Comment by dugenkui03** [comment]
> Can we rename to `ValidationErrorClassification` please

Thanks for your review, `ValidationErrorTypeInterface ` has been rename to `ValidationErrorClassification`.

**Pull request merged**